The openmvg:x86-windows failed with following failures, I have rerun the CI testing.
```
\main_svgSample.dir/manifest.res" failed (exit code 1201) with the following output:
LINK : fatal error LNK1201: error writing to program database 'D:\buildtrees\openmvg\x86-windows-dbg\Windows-AMD64-Debug\main_svgSample.pdb'; check for insufficient disk space, invalid path, or insufficient privilege
ninja: build stopped: subcommand failed.
```

--- a/ports/qt5-base/portfile.cmake
+++ b/ports/qt5-base/portfile.cmake
@@ -135,6 +135,8 @@ endif()
 
 if(WITH_PGSQL)
     list(APPEND CORE_OPTIONS -sql-psql)
+else()
+    list(APPEND CORE_OPTIONS -no-sql-psql)
 endif()
 
 if(VCPKG_TARGET_IS_WINDOWS)

--- a/versions/q-/qt5-base.json
+++ b/versions/q-/qt5-base.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "0baaf03d7141ceb22fea1ff2f4ea7dc5530901df",
+      "git-tree": "a21da1425e7bdf8b22cca4da750b57a6cfc2a3ee",
       "version-string": "5.15.2",
       "port-version": 2
     },

